### ISSUE TITLE
Implementation Dark Theme 

### ISSUE THREAD

**Issue by AhtishamJaved** [hammadsaedi/regex-pro#22]
Dark Mode is not implemented. This tool needs a dark mood like our life.
Thank You

**Comment by AhtishamJaved** [comment]
Please Assign me this Issue
I Love Darkness

**Comment by hammadsaedi** [comment]
Best of luck :)

**Comment by amanpandey3956** [comment]
@hammadsaedi hey pls assign me this issue would love to work on it

**Comment by hammadsaedi** [comment]
taken by @AhtishamJaved

**Comment by AhtishamJaved** [comment]
ok assign him

**Comment by amanpandey3956** [comment]
@AhtishamJaved thank u

**Comment by sfazli96** [comment]
I can give this a try as well if the issue has not been resolved yet. Thank you!

**Comment by hammadsaedi** [comment]
Yes. It should look like this. A button for transition will be a nice addition!
![image](https://github.com/hammadsaedi/regex-pro/assets/71373419/9508cfcb-2ab0-44c1-b052-69c562abfdf3)

**Comment by sfazli96** [comment]
![image](https://github.com/hammadsaedi/regex-pro/assets/43709736/e0cfce80-6779-41c5-830c-024a78c9aa07)
I almost got the dark mode toggle button working, just small things to fix. I just created the pull request, you can check it.
